minigu-cli/src/shell/command.rs

```diff
@@ -45,6 +45,33 @@ pub enum ShellCommand {
         /// If not provided, the current output mode will be printed.
         mode_to_change: Option<OutputMode>,
     },
+
+    /// Set if query metrics should be printed.
+    #[command(name = ":metrics")]
+    Metrics {
+        /// The status to change to.
+        /// If not provided, the current status will be printed.
+        status: Option<Status>,
+    },
+}
+
+#[derive(Debug, Clone, ValueEnum, Display)]
+#[strum(serialize_all = "kebab-case")]
+pub enum Status {
```

Comment:
`status` is too common? may be `ShowMetrics`?

Reply:
I think this is ok since we may need `Status` for future commands? For example, `:header on`.

Reply:
fixed